src/coprocessor/sha256.rs

```diff
@@ -129,6 +131,21 @@ impl<F: LurkField> Sha256Coprocessor<F> {
     }
 }
 
+fn discard_bits<Scalar: LurkField>(bytes: &mut [u8]) {
```

Comment:
Can you add a docstring to this function, please?

Reply:
Yup. "Retains the Scalar::CAPACITY last bits of a big-endian input" would be a possible example.